packages/core/src/2d/sprite/Sprite.ts

```diff
@@ -0,0 +1,260 @@
+import { Vector2, Vector4 } from "@oasis-engine/math";
+import { RefObject } from "../../asset/RefObject";
+import { Engine } from "../../Engine";
+import { Texture2D } from "../../texture";
+
+// A 2D Rectangle defined by X and Y position, width and height.
+export class Rect {
```

Comment:
I think this should put in math folder and named Rect or Rectange

Reply:
> I think this should put in math folder and named Rect or Rectange

done